fuquay-varina.md

```diff
@@ -0,0 +1,5 @@
+## Fuquay Varina, NC
+
+### Places to eat
+
```

Comment:
@crichID I really like hot dogs, can you recommend a good place to get one in Fuquay?